backend/actions/Model/getDocuments.js

```diff
@@ -28,7 +28,7 @@ module.exports = ({ db }) => async function getDocuments(params) {
   params = new GetDocumentsParams(params);
   let { filter } = params;
   console.log('what is filter', filter);
-  if (filter != null) {
+  if (JSON.stringify(filter) !== '{}' && filter != null) {
```

Comment:
Why `JSON.stringify(filter) !== '{}'` rather than just `Object.keys(filter).length > 0` ?

Reply:
It was an issue of placement. It was giving me an error because I was checking for `Object.eys(filter).length > 0` first and so it would throw a `TypeError: cannot convert undefined or null to object`. I've corrected it now so that it checks `filter != null` first